hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -683,6 +691,14 @@
       dfs.container.ratis.datanode.storage.dir be configured separately.
     </description>
   </property>
+  <property>
+    <name>ozone.metadata.dirs.permissions</name>
+    <value>777</value>
```

Comment:
Do default needs to more restrictive 750 or 755 or what's advantage of adding this and other properties below with 777 or full permission?